src/main/java/team/creative/creativecore/common/util/type/list/Tuple.java

```diff
@@ -36,7 +36,7 @@ public int hashCode() {
     public boolean is(K key) {
         if (this.key != null)
             return this.key.equals(key);
-        return this.key == key;
+        return key == null; // FIXME: uhh

```

Comment:
Same as the one before in `Pair`

Reply:
Sure thing, but would be cool if you can remove the // FIXME comment (also for the `Pair` class).

Reply:
done